error/src/lib.rs

```diff
@@ -20,410 +20,365 @@ use core::num::NonZeroU32;
 #[derive(Debug, Copy, Clone, Eq, PartialEq)]
 pub struct CaliptraError(pub NonZeroU32);
 impl CaliptraError {
-    const fn new_const(val: u32) -> Self {
```

Comment:
Are the changes in this file required? I would prefer to avoid including this change in this PR if we don't have to.

Reply:
Sure, I will put it in a separate PR then.